### DISCUSSION TITLE
Modify the INPUT size image of YOLOV8

### DISCUSSION THREAD

**Discussion by Mishaco777** [ultralytics/ultralytics#7086] [Q&A]
Hello, 

How could I modify the first layer to change the input size parameter to set a new resolution of the images that I want to train. I am working with a linear camera an the image that I am constructing have an special size with a high value for width, specify the size of my images is 8192 x 1000 . Also, I want to know if change the input of the YOLOv8 is recommended to improve the performance for my specific problem. Please could you help me. 

Thank you in advance I hope it can be done.

**Comment by pderrenger** [comment]
@Mishaco777 hello! To modify the input image size for training YOLOv8, you can adjust the `imgsz` parameter in your training script. For your specific case with an image size of 8192x1000, you would set `imgsz` to `[8192, 1000]`. Here's an example of how you might do this:

```python
from ultralytics import YOLO

# Load your model configuration and specify the custom image size
model = YOLO('yolov8n.yaml', imgsz=[8192, 1000])

# Train your model with the custom image size
results = model.train(data='your_dataset.yaml', epochs=50)
```

Keep in mind that using such a large image size will significantly increase the computational requirements. It's recommended to have a GPU with a large amount of memory to handle this resolution. If you encounter memory issues, you may need to reduce the batch size or image size, or consider tiling your images into smaller segments.

Changing the input size can potentially improve performance for your specific problem, especially if the default input size does not capture the necessary detail in your images. However, it's important to experiment and validate whether this change indeed yields better results for your use case. Good luck with your project! 😊🚀
